src/components/QuestionItem/QuestionItem.tsx

```diff
@@ -0,0 +1,89 @@
+import { Avatar, Card, CardActions, CardContent, CardHeader, IconButton, Typography } from "@material-ui/core";
+import { red } from "@material-ui/core/colors";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import FavoriteIcon from "@material-ui/icons/Favorite";
+
+import * as React from "react";
+// import { FormattedMessage, InjectedIntl, injectIntl } from "react-intl";
+import { Question } from "../../models/Question";
+
+// import messages from "../../languages/messages";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    card: {
+      width: 100
+    },
+    media: {
+      height: 0,
+      paddingTop: "56.25%" // 16:9
+    },
+    expand: {
+      transform: "rotate(0deg)",
+      marginLeft: "auto",
+      transition: theme.transitions.create("transform", {
+        duration: theme.transitions.duration.shortest
+      })
+    },
+    expandOpen: {
+      transform: "rotate(180deg)"
+    },
+    avatar: {
+      backgroundColor: red[500]
+    }
+  })
+);
+
+interface Props {
+  question: Question;
+  handleOnClick: any;
+  // intl: InjectedIntl;
```

Comment:
You require react-intl to translate time format. See TalkItem and `const startHour`.